### DISCUSSION TITLE
Anyway I can completely uninstall EP in 24H2?

### DISCUSSION THREAD

**Discussion by ZouNuoLan** [valinet/ExplorerPatcher#3318] [Q&A] [answered]
My PC is in insider RP channel and upgrade 24H2. While I first upgrade this patch, the Windows upgrade program noticed me that I must uninstall the EP before continue to upgarde. Then I uninstalled EP and upgrade my PC. The updrade is successful. 
Then I try to install the EP back but failed. The system told me EP installer can not running in this system. I renamed installer from "EP_steup.exe" to "setup.exe" and installed EP successfully. But I can not use Winwdows 10 style start menu because it not avalible on 24H2. 
I decide to uninstall EP but failed again for the same reason. EP installer can not running in this system. I renamed the installer again, from "setup.exe" to "uninstall.exe" and run it. Then it told me I have uninstalled EP sunccessfully. I reboot my PC and login, I still can find EP's menu while I right click the task bar. The EP is still in my PC. I turn to use GEEK, a powerful uninstall program to delete EP's everything include its folder and register. But I can not completely uninstall it. The folder is gone, I reboot my PC again, I still can find EP's menu while I right click the task bar, I click the 'Properties' and find it has an error said something has missing,
I download a new Windows ISO and install it, hope it can fix this but failed.
Maybe reset the system is the only way to fix this problem?

**Comment by pyrates999** [comment, marked as answer]
Disable windows defender when you install and uninstall EP this time.

**Reply by ZouNuoLan** [reply]
Appreciate for your solution. Thanks!
